tests/sentry/tasks/test_merge.py

```diff
@@ -51,6 +51,29 @@ def test_merge_creates_redirect(self):
             group_id=groups[2].id,
         ).count() == 2
 
+    def test_merge_updates_tag_values_seen(self):
+        project = self.create_project()
+        groups = [self.create_group(project) for _ in xrange(0, 2)]
+
+        for group in groups:
```

Comment:
I was going to do this, but to assert against the regression, we need different `GroupTagKey` objects with different `key`s.